recipes/s2let/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "2.2.3":
```

Comment:
I think the version should be 2.2.3rc1. A release candidate is not a final release.
I don't know the s2let release progress though.

Reply:
@mdavezac your last few PRs made made releases when the close to completion... is this the same situation?

Reply:
That is correct. I'll make a full release upstream once this PR is ready.

Reply:
Hi, @mdavezac. I need to say something here: `conan-center-index` is not CI service to build and test projects, we are creating packages for libraries that are already released... here we are not building, nor running library tests.

I would ask you to open the PR once the release is ready, all tests are working and it is ready for production. Every new commit in the PR will invalidate previous reviews/approvals and we will need to go through the PR again and again to review changes and finally merge.

Anyway, we are always happy to welcome new libraries and help the open-source C/C++ community out there 🙌 . Maybe this is a legit use case and/or we can help you in any other way, so please, don't hesitate to share your thoughts with us (you can reach us also in CppLang slack).

Reply:
The upstream has now been released and I've updated the link and sha here.